release notes/v0.54.0.md

```diff
@@ -0,0 +1,304 @@
+    const {done, value} = await parser.next();
+    if (done) {
+        throw new Error("No more rows to read");
+    }
+
+    // We expect the `value` property to be an array of strings, where each string is a field
+    // from the CSV record.
+    console.log(done, value);
+}
+```
+
+</details>
+
+### New `k6 cloud upload` command for uploading test files to the cloud [`#3906`](https://github.com/grafana/k6/pull/3906)
+
+Although we continue to refine and improve the cloud service to handle better the uploading of test files,
+as well as the overall user experience, we've added a new `k6 cloud upload` command that replaces the
+`k6 cloud --upload-only` flag used so far, which is now considered deprecated.
+
+### gRPC module updates driven by contributors
+
+#### New `discardResponseMessage` option
+
+[#3877](https://github.com/grafana/k6/pull/3877) and [#3820](https://github.com/grafana/k6/pull/3820) add a new option for gRPC module `discardResponseMessage` which allows users to discard the messages received from the server.
+
+```javascript
+const resp = client.invoke('main.RouteGuide/GetFeature', req, {discardResponseMessage: true});
+```
+
+This reduces the amount of memory required, and the amount of garbage collection, which reduces the load on the testing machine and can help produce more reliable test results.
+
+Thank you, @lzakharov!
+
+#### New argument `meta` for gRPC's stream callbacks
+
+[#3801](https://github.com/grafana/k6/pull/3801) adds a new argument `meta` to gRPC's stream callback, which handles the timestamp of the original event (e.g. when message has been received).
+
+```javascript
+let stream = new grpc.Stream(client, "main.FeatureExplorer/ListFeatures")
+stream.on('data', function (data, meta) {
+    // will print the timestamp when message has been received
+    call(meta.ts);
+});
+```
+
+Thank you, @cchamplin!
+
+#### Allow missing file descriptors for gRPC reflection
+
+[#3871](https://github.com/grafana/k6/pull/3871) allows missing file descriptors for gRPC reflection.
+
+Thank you, @Lordnibbler!
+
+### Sobek updates brings support of new ECMAScript features into k6 [`#3899`](https://github.com/grafana/k6/pull/3899), [`#3925`](https://github.com/grafana/k6/pull/3925), [`#3913`](https://github.com/grafana/k6/pull/3913)
+
+With this release, we've updated [Sobek](https://github.com/grafana/sobek) (the `ECMAScript` implementation in Go) which contains the new ECMAScript features that are now available in k6.
+
+This includes support for numeric literal separators, like:
+
+```javascript
+const billion = 1_000_000_000
+```
+
+Support for `BigInt`, the values which are too large to be represented by the number primitive.
+
+```javascript
+const huge = BigInt(9007199254740991);
+```
+
+Note: that prior k6 version v0.54, Golang's type `math/big.Int` mapped to another type, so it might be a breaking change for some extensions or users.
+
+Regexp dotall support - letting you match newline characters with `.`.
```

Comment:
```suggestion
Note: Before k6 version v0.54, Golang's type `math/big.Int` mapped to another type, so this might be a breaking change for some extensions or users.

RegExp dotAll support, where you can match newline characters with `.`:
```